src/items/generics.md

```diff
@@ -31,6 +31,7 @@ fn foo<'a, T>() {}
 trait A<U> {}
 struct Ref<'a, T> where T: 'a { r: &'a T }
 struct InnerArray<T, const N: usize>([T; N]);
+struct Uwu<const N: bool, U>(U);
```

Comment:
```suggestion
struct EitherOrderWorks<const N: bool, U>(U);
```

No issue with the pwayfuwness, just twying to document what this example is twying to show. :smiley_cat:

Reply:
Might I suggest `EithewOwdewWowks`